src/views/Components/Components.scss

```diff
@@ -2,6 +2,67 @@
   margin-top: 9% !important;
```

Comment:
Why has `!important` been included everywhere? Is there any other way without enforcing it, this is not a good practice?